src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar  calendar--start-day-sun calendar--month-length-31">
+      <div class="calendar__day"></div>
```

Comment:
Would be better to use semantic html tags
```suggestion
    <ul class="calendar  calendar--start-day-sun calendar--month-length-31">
      <li class="calendar__day"></div>
```